features/bsm/persistence/api/src/main/java/org/opennms/netmgt/bsm/persistence/api/BusinessServiceEdgeEntity.java

```diff
@@ -180,4 +180,9 @@ public boolean equalsDefinition(BusinessServiceEdgeEntity other) {
                 && getMapFunction().equalsDefinition(other.getMapFunction());
         return equals;
     }
+
+    @Override
+    public <T> T accept(EdgeEntityVisitor<T> visitor) {
+        return null; // we MUST overwrite this in all sub-classes properly
```

Comment:
Can we throw a run-time exception instead of returning null? If we can't, can we make a comment?